main.py

```diff
@@ -148,5 +149,76 @@ async def keen(context):
     await context.send(result)
 
 
-bot.run(TOKEN)
+@bot.command(name='daily', help='Will give the daily leaderboard for specified day')
+async def daily(context, day : str = str(time.localtime(time.time())[7] - 335)):
```

Comment:
`datetime.datetime.today().day` would be much cleaner

Reply:
Done.  Much nicer thanks